packages/mongodb-chatbot-server/src/services/ChatLlm.ts

```diff
@@ -4,10 +4,10 @@
  * from that. This interface could still work with non OpenAI providers if they
  * implement the same interface.
  */
-import { ChatMessage, ChatCompletions } from "@azure/openai";
+import { ChatCompletions } from "@azure/openai";
 export type OpenAiMessageRole = "system" | "assistant" | "user";
 
-export interface OpenAiChatMessage extends ChatMessage {
+export interface OpenAiChatMessage {
```

Comment:
Given that this is an OpenAI-specific interface, why not inherit the OpenAI `ChatMessage` interface?

Reply:
they removed it from the latest version of the SDK

Reply:
we were getting a typescript compile err b/c the sdk doesn't export the type any more